README.md

```diff
@@ -169,7 +169,7 @@ To install from source, see [For Developers](#for-developers) section below.
   # Bootstrap estimator wrapper
   boot_est = BootstrapEstimator(est, n_bootstrap_samples=10)
   boot_est.fit(Y, T, X, W)
-  treatment_effect_interval = boot_est.const_marginal_effect_interval(X_test, lower=1, upper=99)
+  treatment_effect_interval = boot_est.effect_interval(X_test, lower=1, upper=99)
```

Comment:
Once #76 is approved (ahem) this example should be further revised since explicitly using a wrapper will no longer be necessary.

Reply:
Sounds good! I will take a look/ approve #76 first and then update this PR accordingly.

Reply:
I just merged #76 so you can make this change now.